README.md

```diff
@@ -22,3 +27,12 @@ officially supported languages have well-supported Avro libraries:
 |PHP|[avro-php](https://github.com/wikimedia/avro-php)|
 |Python|[Apache Avro Python](https://avro.apache.org/docs/current/gettingstartedpython.html)|
 |Ruby|[AvroTurf](https://github.com/dasch/avro_turf)|
+
+## Resources
+
+**Documentation**
+- [Salesforce Developers Blog Post](https://developer.salesforce.com/blogs/2021/07/pub-sub-api-building-event-driven-integrations-just-got-even-easier)
+- [Salesforce Developers Podcast Episode](https://developer.salesforce.com/podcast/2021/08/episode-93-pub-sub-api-with-emmett-chen-ran)
+
+**Additional Code Samples**
+- [Node client](github.com/pozil/pub-sub-api-node-client)
```

Comment:
@pozil This link doesn't work for me. I see "https://github.com/developerforce/pub-sub-api-pilot/blob/pozil/doc-extra-resources/github.com/pozil/pub-sub-api-node-client" when I click on it. Can you add https:// in front?